editor/src/core/third-party/antd-components.ts

```diff
@@ -23,10 +23,14 @@ function createBasicComponent(
 ): ComponentDescriptor {
   const defaultAttributes = getDefaultPropsAsAttributes(propertyControls)
   return componentDescriptor(
-    {
-      antd: importDetails(null, [importAlias(baseVariable)], null),
-      'antd/dist/antd.css': importDetails(null, [], null),
-    },
+    [
+      {
+        source: 'antd',
+        name: baseVariable,
+        type: null,
+      },
+      { source: 'antd/dist/antd.css', name: null, type: null },
+    ],
     jsxElementWithoutUID(jsxElementName(baseVariable, propertyPathParts), defaultAttributes, []),
     name,
```

Comment:
Looks like this `name` is no longer used anywhere after this change (looking at the `project-components.spec` snapshot change)

Reply:
I plan to pretty much delete this system of `DependencyBoundDescriptors`, because insertion will use `EditorState.propertyControlsInfo` and my PR will feed propertyControlsInfo from another (simpler) source.